### ISSUE TITLE
Power are sometimes ignored by astToLatex

### ISSUE THREAD

**Issue by ValentinLeTallec** [arnog/mathlive#564]
## Description

This call :

```typescript
console.log(MathLive.astToLatex({
      fn: 'add',
      arg: [ { sym: 'x' }, { num: '1' } ],
      sup: { num: '2' }
    }));
```
Does not give out the right value.

### Actual behavior

Give out ```x + 1```

### Expected Behavior
Give out ```(x+1)^2```

## Steps to Reproduce

Just call :
```typescript
console.log(MathLive.astToLatex({
      fn: 'add',
      arg: [ { sym: 'x' }, { num: '1' } ],
      sup: { num: '2' }
    }));
```
### Environment

**Operating System**
Ubuntu 20.04 LTS

**Browser**
Firefox

**Comment by arnog** [comment]
This issue will be resolved when #500 lands (this will involve a new MathJSON format). Keeping this open for now to ensure that the equivalent call produce the intended result.

**Comment by arnog** [comment]
This has been fixed with #500.
